src/coreComponents/constitutive/solid/DruckerPrager.hpp

```diff
@@ -0,0 +1,449 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2019 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2019 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2019 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All right reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ *  @file DruckerPrager.hpp
+ */
+
+#ifndef GEOSX_CONSTITUTIVE_SOLID_DRUCKERPRAGER_HPP
+#define GEOSX_CONSTITUTIVE_SOLID_DRUCKERPRAGER_HPP
+
```

Comment:
You should add a "_" at the end of this flag to make it consistent with that at the end of this file as well as the flag in other files.